### PR TITLE
FileSecurity parameter is described as an enum instead of as a class instance

### PR DESCRIPTION
**Pull request by carlossanlop**
Fixes https://github.com/dotnet/dotnet-api-docs/issues/3377

### CODE REVIEW

File: xml/System.IO/File.xml [hunk 1]
```diff
@@ -1192,7 +1192,7 @@
         <param name="path">The path and name of the file to create.</param>
         <param name="bufferSize">The number of bytes buffered for reads and writes to the file.</param>
         <param name="options">One of the <see cref="T:System.IO.FileOptions" /> values that describes how to create or overwrite the file.</param>
-        <param name="fileSecurity">One of the <see cref="T:System.Security.AccessControl.FileSecurity" /> values that determines the access control and audit security for the file.</param>
+        <param name="fileSecurity">An instance of the <see cref="T:System.Security.AccessControl.FileSecurity" /> class that determines the access control and audit security for the file.</param>
```
Comment (mairaw) [inline xml/System.IO/File.xml]: We usually shorten this to objects (except in constructors summaries)

```suggestion
        <param name="fileSecurity">A <see cref="T:System.Security.AccessControl.FileSecurity" /> object that determines the access control and audit security for the file.</param>
```

**Comment by mairaw** [comment]
Removed the milestone info on this to not double count since we have a tracking issue as well

**Comment by carlossanlop** [comment]
Applied @mairaw , thanks. We can merge as soon as the build finishes successfully.
